Formula/vcpkg.rb

```diff
@@ -0,0 +1,40 @@
+class Vcpkg < Formula
+  desc "C++ Library Manager"
+  homepage "https://github.com/microsoft/vcpkg"
+  url "https://github.com/microsoft/vcpkg/archive/2020.04.tar.gz"
+  sha256 "51bdc81d074407b760de5950760f43c253da6cd8135e13744967cf7c95e837fd"
+
+  depends_on "cmake" => :build
+  depends_on "ninja" => :build
+  depends_on "gcc" if MacOS.version <= :mojave
+
+  def install
+    # fix for conflicting declaration of 'char* ctermid(char*)' on Mojave
+    # https://github.com/microsoft/vcpkg/issues/9029
+    ENV.prepend "CXXFLAGS", "-D_CTERMID_H_" if MacOS.version == :mojave
+
+    if MacOS.version <= :mojave
```

Comment:
This conditional block should instead be handled with [Homebrew's built-in compiler selection mechanisms](https://docs.brew.sh/Formula-Cookbook#compiler-selection) if possible.

Reply:
That's cleaner, thanks @alebcay.

Reply:
Thanks for the change - just another note that `depends_on "gcc"` should no longer be needed. The `fails_with` block will cause Homebrew to automatically select a different compiler.

Reply:
Actually, I misspoke, I'm sorry. I think the line _might_ still be needed. I guess just let the CI run and if it fails, just add it back.

Reply:
Looks like gcc was installed and tests are green (even though other formulas seem to have both)

Reply:
Well, I'm glad that worked. I saw the line in other formulas which was why I was concerned that it may still be needed.